CppCoreGuidelines.md

```diff
@@ -3193,9 +3193,9 @@ Informal/non-explicit ranges are a source of errors.
 ##### Note
 
 Ranges are extremely common in C++ code. Typically, they are implicit and their correct use is very hard to ensure.
-In particular, given a pair of arguments `(p, n)` designating an array \[`p`:`p + n`),
+In particular, given a pair of arguments `p, n` designating an array `[p, p + n)`,
 it is in general impossible to know if there really are `n` elements to access following `*p`.
-`span<T>` and `span_p<T>` are simple helper classes designating a \[`p`:`q`) range and a range starting with `p` and ending with the first element for which a predicate is true, respectively.
```

Comment:
note I also put the whole range expression as a code block, instead of just the variables. For rendering the document in many formats (e.g. in html in github), this will look much smoother. Also probably easier to read/edit in markdown.